streaming/src/test/scala/org/apache/spark/streaming/CheckpointSuite.scala

```diff
@@ -133,6 +133,17 @@ trait DStreamCheckpointTester { self: SparkFunSuite =>
     new StreamingContext(SparkContext.getOrCreate(conf), batchDuration)
   }
 
+  /**
+   * Get the first TestOutputStreamWithPartitions, does not check the provided generic type.
+   */
+  protected def getTestOutputStream[V: ClassTag](streams: Array[DStream[_]]):
+    TestOutputStreamWithPartitions[V] = {
+    streams.collect{
```

Comment:
Seems OK. I was going to ask if you can use `find()` here but I don't think it quite applies in the same way. Let's see if Jenkins makes you put a space before the brace